lighthouse-cli/sentry-prompt.ts

```diff
@@ -0,0 +1,67 @@
+import {existsSync} from 'fs';
+import {join as joinPath} from 'path';
+import {inquirer, Configstore} from './shim-modules';
+const log = require('../lighthouse-core/lib/log');
+
+const MAXIMUM_WAIT_TIME = 20 * 1000;
+
+const MESSAGE = [
+  'Lighthouse would like to report back any errors that might occur while auditing. \n  ',
+  'Information such as the URL you are auditing, its subresources, your operating system, Chrome, ',
```

Comment:
this message is pretty huge. (See screenshot) can we break it up?
fwiw the message string [can be colorized](https://github.com/yeoman/insight/blob/a0430e2cb100e45c3d073ca80c7940c165b21f65/lib/index.js#L117) before going into prompt()

> **Lighthouse requests permission to anonymously report back runtime exceptions**
> This can include data such as the test URL, its subresources, your OS, Chrome version, and Lighthouse version. 
> **May _Lighthouse_ report this data to aid in improving the tool?**